reviews/datasource/models.py

```diff
@@ -18,5 +18,6 @@ class PullRequest:
     title: str
     created_at: datetime
     updated_at: datetime
-    approved: bool
+    approved: str
+    rtr: bool
```

Comment:
could we rename this from `rtr` to `approved_by_others` anywhere there is usage?

```suggestion
    approved_by_others: bool
```